api/envoy/extensions/filters/http/buffer/v3/buffer.proto

```diff
@@ -28,6 +28,11 @@ message Buffer {
   // manager will stop buffering and return a 413 response.
   google.protobuf.UInt32Value max_request_bytes = 1
       [(validate.rules).uint32 = {gt: 0}, (validate.rules).message = {required: true}];
+
+  // Set the maximum request size but doesn't buffer the data immediately.
+  // It's useful when the following filters may do the buffer themselves conditionally, so
+  // we won't buffer the data too early.
+  bool set_limit_only = 3;
```

Comment:
What happens if `max_request_bytes` isn't set, is this field ignored?  If so, a comment to that effect would be great.

Reply:
> What happens if `max_request_bytes` isn't set, is this field ignored? If so, a comment to that effect would be great.

https://github.com/envoyproxy/envoy/blob/372a262894f028431cb84f3a1d361fb8fb6a2518/api/envoy/extensions/filters/http/buffer/v3/buffer.proto#L29-L30

The `max_request_bytes` is a required field, so it must be set.